bitnami/harbor/templates/chartmuseum/chartmuseum-dpl.yaml

```diff
@@ -174,12 +184,12 @@ spec:
             - name: internal-tls-certs
               mountPath: /etc/harbor/ssl/chartmuseum
             {{- end }}
-            {{- if .Values.persistence.imageChartStorage.caBundleSecretName }}
+            {{- if .Values.persistence.imageChartStorage.caBundleSecret }}
             - name: storage-service-ca
               mountPath: /harbor_cust_cert/ca-bundle.crt
               subPath: ca.crt
```

Comment:
In other charts we allow setting custom keys, because of integrations with certificates and secret providers like Vault or cert-manager

Reply:
Can we skip this here? Adding it wouldn't add a breaking change and this is PR is already huge enough.

Reply:
Ok but please create a task